apps/meteor/tests/end-to-end/api/teams.ts

```diff
@@ -250,6 +251,36 @@ describe('[Teams]', () => {
 				.expect('Content-Type', 'application/json')
 				.expect(400);
 		});
+
+		it('should delete a team if an error occurs when creating a room for it', async () => {
+			await request
+				.post(api('teams.create'))
+				.set(credentials)
+				.send({
+					name: 'invalid*team*name',
+					type: 0,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(400)
+				.expect((response) => {
+					expect(response.body).to.have.property('success', false);
+					expect(response.body).to.have.property('error');
+					expect(response.body.error).to.be.equal('error-team-creation');
+				});
+
+			await request
```

Comment:
Does it make sense to check for members as well? 🤔

Reply:
I think it makes sense.